test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -28,6 +29,19 @@ def test_ec2_tensorflow_inference_cpu(tensorflow_inference, ec2_connection, regi
     run_ec2_tensorflow_inference(tensorflow_inference, ec2_connection, "8500", region)
 
 
+@pytest.mark.model("half_plus_two")
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_CPU_INSTANCE_TYPE, indirect=True)
+@pytest.mark.parametrize("ei_accelerator_type", TF_EC2_EIA_ACCELERATOR_TYPE, indirect=True)
```

Comment:
nit: it's fairly clear from the name, but can you also add `@pytest.mark.integration("elastic_inference")`